donations/views.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework import viewsets
+from .models import ThankYouNote, DonationPopup
+from .serializers import ThankYouNoteSerializer, DonationPopupSerializer
+from rest_framework.decorators import action
+from django.utils import timezone
+from django.http import JsonResponse
+
+
+class ThankYouNoteViewSet(viewsets.ModelViewSet):
+    serializer_class = ThankYouNoteSerializer
+
+    @action(methods=['post'], detail=True)
+    def post(self, request):
+        thank_you_note = request.data['thank_you_note']
+        user_info = request.data['user_info']
+        created = timezone.now()
```

Comment:
I don't think you need this here since you are using an `auto_now_add` field.

Reply:
@mwharrison When I was testing, the auto update did not work so I added this. I can remove it and see what happens. I could have been doing something wrong.

Reply:
When I manually inserted some test data, I expected the field to be populated (Postgres doing the auto update) and it was not so I added it to the code. I've removed it and the tests are passing.